packages/oui-button/src/button.html

```diff
@@ -0,0 +1,20 @@
+<button
+  arial-label="{{$ctrl.text}}"
+  class="oui-button"
+  ng-attr-id="{{::$ctrl.id}}"
+  ng-attr-name="{{::$ctrl.name}}"
+  ng-attr-type="{{::$ctrl.type}}"
+  ng-class="{
+    'oui-button_primary': $ctrl.variant === 'primary',
+    'oui-button_secondary': $ctrl.variant === 'secondary',
+    'oui-button_link': $ctrl.variant === 'link',
+    'oui-button_icon-left': $ctrl.variantNav === 'previous',
+    'oui-button_icon-right': $ctrl.variantNav === 'next'
+  }"
+  ng-click="$ctrl.ngClick()"
+  ng-disabled="$ctrl.disabled"
+>
+  <span aria-hidden="true" class="oui-icon oui-icon_circle oui-icon-chevron-left" ng-if="$ctrl.variantNav === 'previous'"></span>
+  {{$ctrl.text}}
```

Comment:
[To talk at our next daily] Normally, at OVH we choose to prefer `<span data-ng-bind="$ctrl.text"></span>`. Is this requirement still needed or we can use `{{}}` syntax directly in the html ?